packages/backend-core/src/security/sessions.js

```diff
@@ -14,25 +14,7 @@ function makeSessionID(userId, sessionId) {
   return `${userId}/${sessionId}`
 }
 
-exports.createASession = async (userId, session) => {
-  // invalidate all other sessions
-  await this.invalidateSessions(userId)
-
-  const client = await redis.getSessionClient()
-  const sessionId = session.sessionId
-  if (!session.csrfToken) {
-    session.csrfToken = uuidv4()
-  }
-  session = {
-    createdAt: new Date().toISOString(),
-    lastAccessedAt: new Date().toISOString(),
-    ...session,
-    userId,
-  }
-  await client.store(makeSessionID(userId, sessionId), session, EXPIRY_SECONDS)
-}
-
-exports.invalidateSessions = async (userId, sessionIds = null) => {
+async function invalidateSessions(userId, sessionIds = null) {
```

Comment:
Preference would be to keep the ES6 syntax as backend is all going that way, esp with TS refactor of this package pending

Reply:
That was my recommendation to keep it consistent with the other function declarations in this file which are just plain functions too (the others that aren't directly exported) - but glad to hear the backend is adopting ES6 syntax more, so we could just update the other functions to all be ES6 syntax too.